src/opossum_lib/file_generation.py

```diff
@@ -53,3 +76,17 @@ def tree_to_dict(graph: DiGraph, source: str) -> Union[int, Dict[str, Any]]:
         successor_name = graph.nodes[successor]["label"]
         branch_dict[successor_name] = tree_to_dict(graph, successor)
     return branch_dict
+
+
+def get_breakpoints(graph: DiGraph, source: str) -> List[str]:
```

Comment:
The `graph` is expected to be a tree, right? Maybe rename the parameter `graph` to `tree` to make this clear.

Reply:
Correct, I renamed this, also in `tree_to_dict` above.